.gitignore

```diff
@@ -136,4 +136,5 @@ examples/examples
 */hidden.env
 local/
 pipeline-bak/
-*hidden*.env
\ No newline at end of file
+*hidden*.env
+hidden.env
```

Comment:
Does `*hidden*.env` not match `hidden.env`?

Reply:
it was not ignoring hidden.env
tbh .gitignore sometimes eludes me

Reply:
How about we change this to `*.env`? From what I've seen these are always 'local' files. If examples exist they have different extensions (`*.env.example`) which a dev can copy to `*.env`.